src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -36,7 +37,7 @@ public TestState()
 
             public ActorSystem System { get; set; }
             public TestKitSettings TestKitSettings { get; set; }
-            public ITestQueue<MessageEnvelope> Queue { get; set; }
+            public Channel<MessageEnvelope> Queue { get; set; }
```

Comment:
Replace `TestState.Queue` to use a `Channel<MessageEnvelope>` instead of the old `ITestQueue`